BoxContentSDK/BoxContentSDK/Requests/BOXRequest.m

```diff
@@ -296,7 +296,8 @@ - (NSArray *)fullFileFieldsArray
                        BOXAPIObjectKeyExtension,
                        BOXAPIObjectKeyIsPackage,
                        BOXAPIObjectKeyAllowedSharedLinkAccessLevels,
-                       BOXAPIObjectKeyCollections];
+                       BOXAPIObjectKeyCollections,
+                       BOXAPIObjectKeyRepresentations];
```

Comment:
It's currently not very performant, would definitely consider adding it to exclude fields.